### PR TITLE
16 June - SAP connectors support for OData V4

### PR DESCRIPTION
**Pull request by MarkvanMents**
New release - awaiting confirmation of the date.

### CODE REVIEW

File: content/partners/sap/use-sap-odata-model-creator.md [hunk 1]
```diff
@@ -34,10 +34,6 @@ Before starting this how-to, make sure you have completed the following prerequi
 OData Model Creator for SAP solutions will produce a module which is in Mendix version 7.23.6. To use the module you will have to create your app in, or upgrade your app to, Mendix version 7.23.6 or above.
```
Comment (gauravkrshah) [inline content/partners/sap/use-sap-odata-model-creator.md:34]: Mendix version is upgraded to 8.15.1.

**Review by gauravkrshah — CHANGES_REQUESTED** [review]
Please update supported Mendix version to 8.15.1 or above.

File: content/partners/sap/use-sap-odata-model-creator.md (hunk 1 above)
Comment (MarkvanMents) [reply content/partners/sap/use-sap-odata-model-creator.md:34]: Hi Gaurav,
Will customers on 7.23 still be able to use the old version of the model creator?
We are still supposed to be supporting Mx7 and I hope this is not going to impact any SAP customers who are still on Mx 7 LTS.

Comment (gauravkrshah) [reply content/partners/sap/use-sap-odata-model-creator.md:34]: Hi Mark, 
Upgrade of Mendix version is to support latest Mendix version 9.x. New model created will require Mendix v8.15.1 or above.  They will not be backward compatible. All SAP connectors modules are on the same Mendix version.

Comment (MarkvanMents) [reply content/partners/sap/use-sap-odata-model-creator.md:34]: Thanks for the clarification.

Comment (MarkvanMents) [reply content/partners/sap/use-sap-odata-model-creator.md]: Minimum version updated.

**Review by gauravkrshah — COMMENTED** [review]
Looks Good

**Review by gauravkrshah — APPROVED** [review]

File: content/partners/sap/sap-odata-connector.md [hunk 2]
```diff
@@ -17,10 +17,6 @@ The OData Connector for SAP solutions is an OData connector written specifically
 
 You can use the [OData Model Creator for SAP solutions](https://sapodatamodelcreator.mendixcloud.com/) to generate a data model from the metadata of your OData service, exposed from your SAP back-end system, or from APIs in the SAP API Business Hub or the SAP Catalog Service. For more details see [How to Use the OData Model Creator for SAP Solutions](/partners/sap/use-sap-odata-model-creator).
 
-{{% alert type="warning" %}}
```
Comment (mayankmendix) [inline content/partners/sap/sap-odata-connector.md:20]: We are also releasing SAPODataConnector also and adding support for OData V4 (only GET calls).
For this kindly add here something like "OData version 4 support is added from OData Connector for SAP solutions v5.3.0, Only for GET functions".

File: content/partners/sap/use-sap-odata-connector.md [hunk 3]
```diff
@@ -16,10 +16,6 @@ The OData Connector for SAP solutions is a specific SAP connector for integratin
 * Configure the OData Connector for SAP solutions
 * Display product data from the SAP OData service GWSAMPLE_BASIC in your Mendix app
 
-{{% alert type="warning" %}}
```
Comment (mayankmendix) [inline content/partners/sap/use-sap-odata-connector.md:19]: Same as above.

File: content/releasenotes/developer-portal/sap-cloud-platform.md [hunk 4]
```diff
@@ -12,6 +12,16 @@ For information on the current status of deployment to SAP BTP and any planned r
 
 ## 2021
```
Comment (mayankmendix) [inline content/releasenotes/developer-portal/sap-cloud-platform.md:14]: Kindly also add release notes for SAP OData Connector as well for OData V4 support (for GET functions).

**Comment by MarkvanMents** [comment]
@mayankmendix 
Hi Mayank,
I've updated the documents as requested in this last commit (https://github.com/mendix/docs/pull/3427/commits/c0ec0a3d0b44e73db99e88d5f03b3fd126d8e584).
Please let me know when you are happy and I can publish.
Thanks.

**Comment by MarkvanMents** [comment]
Changes approved over Slack.

**Pull request merged**